jobs/integration/conftest.py

```diff
@@ -150,13 +150,24 @@ async def run(self, cmd, *args, stdin=None):
             )
         return stdout.decode("utf8"), stderr.decode("utf8")
 
-    async def juju_wait(self, *args, **kwargs):
-        cmd = ["/snap/bin/juju-wait", "-e", self.connection, "-w", "-v"]
-        if args:
-            cmd.extend(args)
-        if "timeout_secs" in kwargs and kwargs["timeout_secs"]:
-            cmd.extend(["-t", str(kwargs["timeout_secs"])])
-        return await self.run(*cmd)
+    def juju_wait(self, *args, **kwargs):
+        """Run juju-wait command with provided arguments.
+
+        method is defined as async -- but it isn't necessary
+        as the latest implementation uses `sh` and doesn't
+        support async/await.
+
+        if kwarg contains `m`: juju-wait is executed on a different model
+        see juju-wait --help for other supported arguments
+        """
+
+        command = sh.Command("/snap/bin/juju-wait")
+        if "m" not in kwargs:
+            kwargs["m"] = self.connection
```

Comment:
`-e` and `-m` are the same command switch.  I've replaced with `m` to closer match `juju` command and to pick up certain `juju_wait(...)` calls that were supposed to be waiting on a different model